_docs-sources/pipelines/production-use/index.md

```diff
@@ -0,0 +1,27 @@
+# Production Use
+
+In the [Hello World Tutorial](../hello-world), we got Pipelines up and running for the first time. In this tutorial, we make it production-ready.
+
+## Branch protection rules
```

Comment:
This is duplicate work, I've already scaffolded this out here https://github.com/gruntwork-io/docs/pull/1116

Reply:
Actually, I think they are related but independent. The key business need here is to give customers step-by-step instructions for getting Pipelines ready for prod, including specific, opinionated branch protection rule recommendations (or calling out the options and tradeoffs among them).

By contrast, #1116 seems focused on explaining the mechanisms we use to achieve a strong security posture. 

To make the distinction more concrete, I could imagine the DevOps engineer tasked with setting up Pipelines referring to this "Production use" guide, while the Security engineer tasked with vetting Pipelines as acceptable to install in the first place, would review #1116.

Reply:
I recently updated #1116 to be more limited in scope after seeing a different ticket in the queue for documenting the token creation bits to reduce the potential for duplicate content, so I'm fine with distinguishing the two.

The only difference between the tutorial and a prod setup for pipelines is creating more tokens that have limited scope. I suspect this doc will be quite short.

Reply:
Added in #1126

Reply:
Roger, and thanks!